source/containers.tex

```diff
@@ -16086,6 +16093,1215 @@
+contain no equal elements.
+
+\pnum
+\effects
+Equivalent to: \tcode{c = std::move(cont);}
+\end{itemdescr}
+
+\rSec3[flat.set.erasure]{Erasure}
+
+\indexlibrarymember{erase_if}{flat_set}%
+\begin{itemdecl}
+template<class Key, class Compare, class KeyContainer, class Predicate>
+  size_t erase_if(flat_set<Key, Compare, KeyContainer>& c, Predicate pred);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto [erase_first, erase_last] = ranges::remove_if(c, pred);
+auto n = erase_last - erase_first;
+c.erase(erase_first, erase_last);
+return n;
+\end{codeblock}
+\end{itemdescr}
+
+\rSec2[flat.multiset]{Class template \tcode{flat_multiset}}
+
+\rSec3[flat.multiset.overview]{Overview}
+
+\pnum
+\indexlibraryglobal{flat.multiset}%
+A \tcode{flat_multiset} is a container adaptor
+that provides an associative container interface
+that supports equivalent keys
+(i.e., possibly containing multiple copies of the same key value) and
+provides for fast retrieval of the keys themselves.
+\tcode{flat_multiset} supports iterators that model the
+\libconcept{random_access_iterator} concept\iref{iterator.concept.random.access}.
+
+\pnum
+A \tcode{flat_multiset} meets all of the requirements
+for a container\iref{container.reqmts} and
+for a reversible container\iref{container.rev.reqmts},
+plus the optional container requirements\iref{container.opt.reqmts}.
+\tcode{flat_multiset} meets the requirements of
+an associative container\iref{associative.reqmts}, except that:
+\begin{itemize}
+\item
+it does not meet the requirements
+related to node handles\iref{container.node.overview},
+\item
+it does not meet the requirements related to iterator invalidation, and
+\item
+the time complexity of the operations
+that insert or erase a single element from the
+set is linear,
+including the ones that take an insertion position iterator.
+\end{itemize}
+\begin{note}
+A \tcode{flat_multiset} does not meet
+the additional requirements of an allocator-aware container,
+as described in \ref{container.alloc.reqmts}.
+\end{note}
+
+\pnum
+A \tcode{flat_multiset} also provides most operations
+described in \ref{associative.reqmts} for equal keys.
+This means that a \tcode{flat_multiset} supports
+the \tcode{a_eq} operations in \ref{associative.reqmts},
+but not the \tcode{a_uniq} operations.
+For a \tcode{flat_mutliset<Key>},
+both the \tcode{key_type} and \tcode{value_type} are \tcode{Key}.
+
+\pnum
+Descriptions are provided here only for operations on \tcode{flat_multiset}
+that are not described in one of the general sections or
+for operations where there is additional semantic information.
+
+\pnum
+A \tcode{flat_multiset} maintains the invariant
+that the keys are sorted with respect to the comparison object.
+
+\pnum
+If any member function in \ref{flat.multiset.defn} exits via an exception,
+the invariant is restored.
+\begin{note}
+This cat result in the \tcode{flat_multiset}'s being emptied.
```

Comment:
```suggestion
This can result in the \tcode{flat_multiset} being emptied.
```

Reply:
The second thing is good as-is, the typo fix is of course welcome.

Reply:
```suggestion
This can result in the \tcode{flat_multiset}'s being emptied.
```